src/app/components/organization-switcher.component.html

```diff
@@ -0,0 +1,66 @@
+<div *ngIf="loaded && activeOrganization != null" class="tw-flex">
+  <button
+    class="tw-flex tw-items-center tw-bg-background-alt tw-border-none"
+    type="button"
+    id="pickerButton"
+    [appA11yTitle]="'organizationPicker' | i18n"
+    [bitMenuTriggerFor]="orgPickerMenu"
+  >
+    <app-avatar
+      [data]="activeOrganization.name"
+      size="45"
+      [circle]="true"
+      [dynamic]="true"
+    ></app-avatar>
+    <div class="org-name tw-ml-3">
+      <span>{{ activeOrganization.name }}</span>
+      <small class="tw-text-muted">{{ "organization" | i18n }}</small>
+    </div>
+    <div class="tw-ml-3">
+      <i class="bwi bwi-angle-down tw-text-main" aria-hidden="true"></i>
```

Comment:
The arrow on this switcher looks a little strange to me with how it's centered on the entire switcher itself and floats lower than the org name. Do you think it look better in line with the organization name?

Reply:
I think either would be fine, but I think the Figma agrees with you:
![Screen Shot 2022-05-05 at 1 40 58 pm](https://user-images.githubusercontent.com/31796059/166860659-50945ad0-3f7e-473d-b15d-c91ff2b62156.png)

I'll fix this shortly.

Reply:
Done, here's the updated version:
![Screen Shot 2022-05-05 at 2 16 01 pm](https://user-images.githubusercontent.com/31796059/166863268-d7f39ff7-611b-4027-9535-f4eb62c65714.png)

Reply:
Update: no truncation required